--- libgit2-glib-0.26.4/libgit2-glib/ggit-config.c~	2018-03-02 10:51:47.000000000 +0100
+++ libgit2-glib-0.26.4/libgit2-glib/ggit-config.c	2018-09-20 18:11:04.838368107 +0200
@@ -295,6 +295,7 @@ ggit_config_add_file (GgitConfig      *c
 	ret = git_config_add_file_ondisk (_ggit_native_get (config),
 	                                  path,
 	                                  (git_config_level_t)level,
+	                                  NULL,
 	                                  force);
 	g_free (path);
 
