--- libgit2-glib-0.0.6/libgit2-glib-1.0.pc.in.orig	2012-08-15 12:47:17.000000000 +0200
+++ libgit2-glib-0.0.6/libgit2-glib-1.0.pc.in	2013-07-24 21:26:27.542630804 +0200
@@ -5,7 +5,7 @@
 
 Name: libgit2-glib
 Description: libgit2-glib, a a glib wrapper library around the libgit2 git access library.
-Requires: libgit2 >= GIT2_REQUIRED, glib-2.0 >= @GLIB_REQUIRED@, gobject-2.0 >= @GLIB_REQUIRED@, gio-2.0 >= @GLIB_REQUIRED@, gobject-introspection-1.0 >= @INTROSPECTION_REQUIRED
+Requires: libgit2 >= @GIT2_REQUIRED@, glib-2.0 >= @GLIB_REQUIRED@, gobject-2.0 >= @GLIB_REQUIRED@, gio-2.0 >= @GLIB_REQUIRED@, gobject-introspection-1.0 >= @INTROSPECTION_REQUIRED@
 Version: @VERSION@
 Cflags: -I${includedir}/libgit2-glib-1.0
 Libs: -L${libdir} -lgit2-glib-1.0
